tests/integration/modules/molecule/pipe_handler/Dockerfile.j2

```diff
@@ -0,0 +1,14 @@
+# Molecule managed
```

Comment:
This should be a symlink pointing to `../shared/Dockerfile.j2`

Reply:
Hmm every time I run molecule, new Dockerfile.j2 is generated and it overwrites the symlink

Reply:
Actually it doesn't overwrite the symlink... my bad. Molecule fails to bring up docker containers if the file is missing.

Reply:
Huh, you got me on this one @aljazkosir because the symlink never got overwritten in my PC and I was wondering if I was doing it wrong :) For now, please just convert it to symlink. In the future, molecule will probably become smart enough to use default for this file (ahem, @tadeboro, when "in the future" depends on you I'm afraid 😄 )